e2e/internal/e2e/keyexec.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package e2e
```

Comment:
This file is added here to provide some `singularity key [...]` commands.

Reply:
Does it belong in e2e/internal/e2e, though? It's very specific to key testing.